cloudconfig/userdatacfg_unix.go

```diff
@@ -122,6 +126,20 @@ func (w *unixConfigure) ConfigureBasic() error {
 			`sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers`,
 		)
 		w.addCleanShutdownJob(service.InitSystemSystemd)
+	case os.OpenSUSE:
+		w.conf.AddScripts(
+			// Mask and stop firewalld, if enabled, so it cannot start. See
+			// http://pad.lv/1492066. firewalld might be missing, in which case
+			// is-enabled and is-active prints an error, which is why the output
+			// is surpressed.
+			"systemctl is-enabled firewalld &> /dev/null && systemctl mask firewalld || true",
+			"systemctl is-active firewalld &> /dev/null && systemctl stop firewalld || true",
+			`sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers`,
+			//Scripts assume ubuntu group for ubuntu user...
+			`(grep ubuntu /etc/group) || groupadd ubuntu`,
```

Comment:
OK. cloud-init does support adding new groups, but we don't expose or use it in Juju at the moment. I may change this later to do it through cloud-init -- this is fine for now.